database_transformations/product.py

```diff
@@ -32,29 +39,33 @@ async def insert_product(columns: str) -> None:
     await Database.execute_many(query, entries)
 
 
-async def get_recommended_products(req: Optional[object]) -> dict:
+async def get_recommended_products(req: Optional[object] = None) -> dict:
     # Create table if it doesn't exist
+
     query = f"""
-    SELECT * FROM Product
-    WHERE id='66faa130bedf3403197df77c'            
+    SELECT * FROM Product            
     ;"""
-    row = dict((await Database.fetchrow(query)).items())
-    row['userrating'] = json.loads(row['userrating'])
-    data = {
-        'justForYou': row,
-        'bestPick': row,
-        'bannerImageUrl': "https://"
-    }
-    return  data
-
-async def get_preference_product_detail(req: Optional[object]) -> dict:
+    rows = await Database.fetch(query)
+
+    dict_data = [dict(row) for row in rows]
+
+    for row in dict_data:
+        row["userrating"] = json.loads(row["userrating"])
+
+    return dict_data
```

Comment:
this should not be select * 
and we should not have for loop in stuff like this

please put this back like it was